dotnet/Cucumber.Messages.Specs/Cucumber.Messages.Specs.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>

```

Comment:
This is just our internal test project, not a deliverable. So this will not cause any issues. (But the other changes will be breaking anyway for .NET, I guess).